css/components/view-modes/highlighted-small/highlighted-small.css

```diff
@@ -63,9 +63,26 @@
 .node--view-mode-highlighted-small .read-more a {
   text-decoration: none;
   text-transform: uppercase;
+  display: inline-block;
+  position: relative;
+  padding-right: 20px;
 }
 
 .node--view-mode-highlighted-small .read-more a:focus,
 .node--view-mode-highlighted-small .read-more a:hover {
   text-decoration: underline;
 }
+
+.node--view-mode-highlighted-small .read-more a:before {
+  content: '';
+  position: absolute;
+  top: 50%;
+  right: 0;
+  width: 14px;
+  height: 14px;
+  margin-top: -7px;
+  background-image: url('../../../../assets/images/svg/pointer.svg');
```

Comment:
Is it possible to avoid defining relative path somehow? This does not feel to be too flexible…

Reply:
One to discuss as a separate issue? Am I correct in thinking that this is a discussion point between what we want to support between:
a) the theme being moved to a new location in a Drupal installation by someone (relative works) or
b) we use absolute path to make it simpler to reference in different files?

However, perhaps I am misunderstanding your point :)